### ISSUE TITLE
Print out libsecp256k1 unit test output in travis

### ISSUE THREAD

**Issue by sipa** [bitcoin/bitcoin#8429]
Today I saw the follow Travis error: https://travis-ci.org/bitcoin/bitcoin/jobs/148435469 (from PR #8393). It has a segfault in libsecp256k, but I can't reproduce because the random unit test seed is not included in the Travis output.

@theuni Can you have a look?

**Comment by theuni** [comment]
Hmm, this was supposed to be fixed by 401ae654b247d80e29e76ebd094e9e37409941bd. Looking into it.

**Comment by fanquake** [comment]
Has this been fixed now? This is the current libsecp256k1 Travis output.
```
============================================================================
Testsuite summary for libsecp256k1 0.1
============================================================================
# TOTAL: 1
# PASS:  1
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
make[6]: Leaving directory `/home/travis/build/bitcoin/bitcoin/build/src/secp256k1'
make[5]: Leaving directory `/home/travis/build/bitcoin/bitcoin/build/src/secp256k1'
make[4]: Leaving directory `/home/travis/build/bitcoin/bitcoin/build/src/secp256k1'
make[4]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/src/univalue'
  CXX      test/test_unitester-unitester.o
  CXXLD    test/unitester
make  check-TESTS
make[5]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/src/univalue'
make[6]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/src/univalue'
PASS: test/unitester
make[7]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/src/univalue'
make  all-am
make[8]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/src/univalue'
make[8]: Leaving directory `/home/travis/build/bitcoin/bitcoin/build/src/univalue'
make[7]: Leaving directory `/home/travis/build/bitcoin/bitcoin/build/src/univalue'
```
